### ISSUE TITLE
Abandon package

### ISSUE THREAD

**Issue by nicolas-grekas** [dunglas/symfony-lock#5]
Thanks to symfony/flex#409
Similar to https://github.com/symfony/lts/pull/13 and https://github.com/symfony/force-lowest/pull/2

**Comment by deflock** [comment]
@nicolas-grekas Not sure. You have to use flex plugin, right? What if using standalone packages and not flex? Is it somehow possible to extract "requiring" functionality from flex plugin to a separate one?

**Comment by nicolas-grekas** [comment]
Yes, it's possible to use flex only for that purpose, see https://github.com/symfony/force-lowest/pull/2#issuecomment-413131983
